Toggl.Foundation.MvvmCross/ViewModels/LoginViewModel.cs

```diff
@@ -35,12 +35,12 @@ public enum ShakeTargets
         private readonly IAnalyticsService analyticsService;
         private readonly IOnboardingStorage onboardingStorage;
         private readonly IMvxNavigationService navigationService;
-        private readonly IPasswordManagerService passwordManagerService;
         private readonly IErrorHandlingService errorHandlingService;
         private readonly ILastTimeUsageStorage lastTimeUsageStorage;
         private readonly ITimeService timeService;
         private readonly ISchedulerProvider schedulerProvider;
         private readonly IRxActionFactory rxActionFactory;
+        public readonly IPasswordManagerService PasswordManagerService;
```

Comment:
We don't do public fields, we do public properties.
```suggestion
        public IPasswordManagerService PasswordManagerService { get; }
```